### DISCUSSION TITLE
[Bug] Running spring boot projects with SW agent in idea causes JMX to be unavailable

### DISCUSSION THREAD

**Discussion by niuone-cloud** [apache/skywalking#12788] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

Running spring boot projects with SW agent in idea causes JMX to be unavailable

### What you expected to happen

SW agent is expected to be JMX compatible

### How to reproduce

idea 2024.2
jdk 21
spring boot 3.x
sw agent 9.3

Either removing SW agent or manually configuring JMX is feasible

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Generally, we don't recommend running the agent in your IDEA unless you want to debug the agent codes themselves.
